packages/sdk/src/utils.ts

```diff
@@ -182,8 +182,8 @@ const generateShareFilename = /* @__PURE__ */ (
 
 const getResourceUrl = (module: ModuleInfo, sourceUrl: string): string => {
   if ('getPublicPath' in module) {
-    const publicPath = new Function(module.getPublicPath)();
-    return `${publicPath}${sourceUrl}`;
+    const publicPath = new Function('return ' + module.getPublicPath)();
```

Comment:
@zhoushaw  This may have conflict . Because getPublicPath and publicPath only appear one , but now the getPublicPath need to pass publicPath as parameter , so can we need to record both fields ?

Reply:
i have added backward compat to the implementation